api-reference/beta/api/group_delta.md

```diff
@@ -38,8 +34,7 @@ This method supports OData Query Parameters to help customize the response.
 _id_ property is always returned. 
 - Delta query support `$select`, `$top`, and `$expand` for groups. 
 - There is limited support for `$filter` and `$orderby`:
-  * The only supported `$filter` expresssions are `$filter=receivedDateTime+ge+{value}` 
-  or `$filter=receivedDateTime+gt+{value}`.
+  * Supported `$filter` expressions are `$filter=receivedDateTime+ge+{value}`, `$filter=receivedDateTime+gt+{value}` and `$filter=id+eq+{value}`.
```

Comment:
Filters are not supported for users and groups in v1.0.
Filter is supported for users and groups in Beta (but only on id).  Jeff can provide example.